src/Main.java

```diff
@@ -2661,7 +2753,11 @@ else if (location == 2) {
                 }
             }
             else if (region == 0x9) {
-                return inventory.contains(Items.JUMP_BOOTS) || inventory.contains(Items.FLUTE);
+				/*
+				* Added MINOR GLITCHES Logic for the Tree Interior Region.
+				* Use a dashjump wallclip to reach the ledge leading to the doors.
+				*/
+                return inventory.contains(Items.JUMP_BOOTS) || inventory.contains(Items.FLUTE) || difficulty >= Difficulty.S_HARD;
```

Comment:
Is this an offscreen wallclip? I've been resistant to adding this to MG difficulty given you can't use a visual cue, unless I'm misunderstanding how this works

Reply:
This is an offscreen jump and is mostly blind, but the pixel alignment is something you can see, and is also consistent. The timing of the right input immediately into pause is just precise. Obviously you're holding left as you unpause. It's not unreasonable to get within a few tries though.

Reply:
Alright, this'll probably be fine. Thanks for the explanation